app/graphql/types/outputs/tube_type.rb

```diff
@@ -4,8 +4,8 @@ module Types
   module Outputs
     # The type for Tube objects.
     class TubeType < CommonOutputObject
-      field :barcode, String, null: false
-      field :materials, [MaterialUnionType], null: true
+      field :barcode, String, 'The barcode of this tube.', null: false
+      field :materials, [MaterialUnionType], 'The material contained in this tube.', null: true
```

Comment:
Worth pluralising _material_ in the description?

Reply:
Oops yes